### PR TITLE
Fix server side rendering issues with search page filter controls

### PR DESCRIPTION
**Pull request by bertyhell**
https://meemoo.atlassian.net/issues/ARC-2197

image
image

### CODE REVIEW

File: src/modules/shared/components/Html/Html.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ export interface HtmlProps {
 const Html: FunctionComponent<HtmlProps> = ({
 	content,
 	sanitizePreset = RICH_TEXT_SANITIZATION,
-	type = 'p',
+	type = 'div',
```
Comment (ian-emsens-shd) [inline src/modules/shared/components/Html/Html.tsx:18]: Make sure to check styles of components that implement `Html` for any selectors relying on the previous `p`-tag default.

File: src/modules/visitor-space/components/NoServerSideRendering/NoServerSideRendering.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { FC, ReactNode, useEffect, useState } from 'react';
+
+interface NoSsrProps {
+	children: ReactNode;
+}
+
+export const NoServerSideRendering: FC<NoSsrProps> = ({ children }) => {
```
Comment (ian-emsens-shd) [inline src/modules/visitor-space/components/NoServerSideRendering/NoServerSideRendering.tsx:7]: Does this component replace `import { isServerSideRendering } from '@shared/utils'` entirely?
If so, probably should take this opportunity to deprecate / refactor instances of `isServerSideRendering`.

**Review by ian-emsens-shd — APPROVED** [review]

File: src/modules/visitor-space/components/NoServerSideRendering/NoServerSideRendering.tsx (hunk 2 above)
Comment (bertyhell) [reply src/modules/visitor-space/components/NoServerSideRendering/NoServerSideRendering.tsx:7]: no, NoServerSideRendering only for rendering things
the isServerSideRendering() is still usefull for switching in logic not related to rendering

**Comment by bertyhell** [comment]
merged in release/sprint-2

**Pull request closed without merging**